lib/services/sku-pack-service.js

```diff
@@ -51,6 +60,151 @@ function skuPackServiceFactory(
     }
 
     /**
+     * Get list of skus
+     * @param {Object} query [req.query] HTTP request
```

Comment:
A little concern for the comment, Since the SkuPackService is just a collection of some util functions for SKU pack, it may not always used by the HTTP APIs, so I think the query doesn't have to be strict from HTTP request,